src/styles/index.scss

```diff
@@ -1,3 +1,258 @@
+  color: #00acdc;
+}
+
+/* #endregion */
+
+/* #region stars */
+.stars {
+  display: flex;
+  width: 100%;
+  max-width: 250px;
+  justify-content: flex-start;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    width: 16px;
+    height: 16px;
+    background-size: cover;
+    margin-right: 4px;
+  }
+
+  &--4 .stars__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
+}
+
+/* #endregion */
+
+/* #region main */
+main {
+  margin-top: 50px;
+}
+
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 46px;
+  margin-bottom: 48px;
+  justify-items: center;
+}
+
+.card {
+  width: 200px;
+  height: auto;
+  padding: 16px;
+  border-radius: 5px;
+  color: $main-text-color;
+  box-sizing: border-box;
+  transition:
+    transform 0.3s ease,
+    box-shadow 0.3s ease;
+  position: relative;
+
+  &:hover {
+    transform: scale(1.2);
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+  }
+
+  &__title:hover {
+    color: #34568b;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__describe {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+    margin-top: 40px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color;
+    margin: 0;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    text-align: center;
+
+    &__reviews {
+      color: $main-text-color;
+      font-size: 12px;
+      line-height: 14px;
+      width: 400px;
+      margin: 0;
+      text-align: right;
+    }
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    line-height: 18px;
+    height: 18px;
+    width: 100%;
+    margin-bottom: 16px;
+    margin-top: 24px;
+
+    &-text {
+      color: $secondary-color;
+      font-size: 12px;
+      margin: 0;
+    }
+
+    h2 {
+      font-weight: 700;
+      font-size: 16px;
+      margin: 0;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid $button-color;
+    border-radius: 5px;
+    width: 164px;
+    height: 38px;
+    color: $button-text;
+    background-color: $button-color;
+    line-height: 16px;
+    text-align: center;
+    font-weight: 700;
+    font-size: 14px;
+    text-transform: uppercase;
+    text-decoration: none;
+    transition:
+      background-color 0.3s ease,
+      color 0.3s ease;
+
+    &:hover {
+      background-color: $button-text;
+      border-color: $button-color;
+      color: $button-color;
+    }
+  }
+}
+
+@media (max-width: 1200px) {
+  .catalog {
+    grid-template-columns: repeat(
+      3,
+      1fr
+    ); // 3 колонки на екранах середнього розміру
+  }
+}
+
+@media (max-width: 768px) {
+  .catalog {
+    grid-template-columns: repeat(2, 1fr); // 2 колонки на малих екранах
+  }
+}
+
+@media (max-width: 480px) {
```

Comment:
The media query breakpoint is set to `480px`. According to the task requirements, it should be `488px`.